sdk-utils/src/main/kotlin/com/rakuten/tech/mobile/sdkutils/PreferencesUtil.kt

```diff
@@ -0,0 +1,147 @@
+package com.rakuten.tech.mobile.sdkutils
+
+import android.content.Context
+
+/**
+ * Helper class to get/set values in shared preferences.
+ * Should be accessed via [PreferencesUtil.getInstance].
+ */
+@SuppressWarnings("TooManyFunctions")
+class PreferencesUtil private constructor(context: Context, name: String) {
+    private val prefs = context.getSharedPreferences(name, Context.MODE_PRIVATE)
+
+    /**
+     * Get integer value from shared preferences.
+     *
+     * @param key the preference key
+     * @return return the integer value
+     */
+    fun getInt(key: String) = prefs.getInt(key, -1)
```

Comment:
add default value parameter with default value of -1, in case user didn't provide any. Please apply to others, null or empty string for non-numeric type

Reply:
Updated